### PR TITLE
Make iSCSI Login and Logout wait for DBus objects update

### LINKED ISSUE #342: Make ISCSI.Initiator.Login/Logout methods really synchronous
These methods should wait for:

* the device to be added/removed to/from the system
* the session objects to be added/removed from DBus (when supported)

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Login and Logout methods now wait for the device and session
objects to be added to (removed from) DBus.

Fixes #342

### CODE REVIEW

File: modules/iscsi/udiskslinuxmanageriscsiinitiator.c [hunk 1]
```diff
@@ -657,6 +724,34 @@ handle_login (UDisksManagerISCSIInitiator *object,
       goto out;
     }
 
+  /* sit and wait until the device appears on dbus */
+  if (udisks_daemon_wait_for_object_sync (manager->daemon,
+                                          wait_for_iscsi_object,
+                                          g_strdup (arg_name),
```
Comment (vpodzime) [inline modules/iscsi/udiskslinuxmanageriscsiinitiator.c]: Something has to free this duplicate string.

Comment (vojtechtrefny) [reply modules/iscsi/udiskslinuxmanageriscsiinitiator.c]: `g_free` below should do that (4th parameter is free function for "user data")

Comment (vpodzime) [reply modules/iscsi/udiskslinuxmanageriscsiinitiator.c]: Oh, okay then. Great! :)

**Comment by dwlehman** [comment]
Does this mean we can remove `blivet.iscsi.iSCSI.stabilize`?

**Pull request merged**